### DISCUSSION TITLE
ialarm-mqtt - no avaiable connection to the central

### DISCUSSION THREAD

**Discussion by Lollo9191** [maxill1/ialarm-mqtt#47] [Q&A]
Hello,
I'm tring to deploy the the integration to connect CasaSicura Alarm central model SiquraWeb.
I'm already blocked to the first step. Seems that the central hasn't the 18034 TCP port opened.
Seems the only ports in listen state are 80,21,22.
What can I do? This behavoiur depends from the firmware version?
@maxill1 
Thanks

**Comment by maxill1** [comment]
> Hello,
> I'm tring to deploy the the integration to connect CasaSicura Alarm central model SiquraWeb.
> I'm already blocked to the first step. Seems that the central hasn't the 18034 TCP port opened.
> Seems the only ports in listen state are 80,21,22.
> What can I do? This behavoiur depends from the firmware version?
> @maxill1 
> Thanks 

If I remember correctly some panel uses port 8000. 
Let's wait some sicuraweb users for firmware suggestions.

**Reply by Lollo9191** [reply]
Thank you!!! Using the port 8000 the connection is working fine!
